filters.py

```diff
@@ -10,4 +10,8 @@
 
 
 def message_filter(message):
+    # If the message has no text, we just ignore it.
+    if message.text is None:
```

Comment:
I think it would be better to test like this `if not message.text:` because empty strings are not evaluated as None type.

```
>>> '' is None
False
>>> not ''
True
```